swagger/components/Hero.js

```diff
@@ -18,8 +18,9 @@ const copyToClipboard = text => {
 
 class Hero extends Component {
   render() {
+    const projects = require("../projects.js");
```

Comment:
I'd really add a comment here that states that those changes should be retained during swaggerui upgrades.